tests/integration/test_helpers.go

```diff
@@ -0,0 +1,109 @@
+package integration
+
+import (
+	"fmt"
+	"testing"
+
+	tmrand "github.com/cometbft/cometbft/libs/rand"
+	tmproto "github.com/cometbft/cometbft/proto/tendermint/types"
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	"github.com/cosmos/cosmos-sdk/crypto/keys/secp256k1"
+	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	minttypes "github.com/cosmos/cosmos-sdk/x/mint/types"
+	"github.com/cosmos/cosmos-sdk/x/staking"
+	stakingtestutil "github.com/cosmos/cosmos-sdk/x/staking/testutil"
+	"github.com/stretchr/testify/require"
+
+	ojoapp "github.com/ojo-network/ojo/app"
+	appparams "github.com/ojo-network/ojo/app/params"
+	airdropkeeper "github.com/ojo-network/ojo/x/airdrop/keeper"
+	airdroptypes "github.com/ojo-network/ojo/x/airdrop/types"
+	oraclekeeper "github.com/ojo-network/ojo/x/oracle/keeper"
+	oracletypes "github.com/ojo-network/ojo/x/oracle/types"
+)
+
+const (
+	bondDenom    = appparams.BondDenom
+	initialPower = int64(1000)
+	isCheckTx    = false
+)
+
+var validatorPowers = []int64{599, 398, 2}
+
+type TestValidatorKey struct {
+	PubKey     cryptotypes.PubKey
+	ValAddress sdk.ValAddress
+	AccAddress sdk.AccAddress
+	Power      int64
+}
+
+func CreateTestValidatorKeys(numValidators int) []TestValidatorKey {
+	var validatorKeys []TestValidatorKey
+
+	for i := 0; i < numValidators; i++ {
+		pubKey := secp256k1.GenPrivKey().PubKey()
+		valInfo := TestValidatorKey{
+			PubKey:     pubKey,
+			ValAddress: sdk.ValAddress(pubKey.Address()),
+			AccAddress: sdk.AccAddress(pubKey.Address()),
+			Power:      validatorPowers[i],
+		}
+		validatorKeys = append(validatorKeys, valInfo)
+	}
+
+	return validatorKeys
+}
+
+func SetupAppWithContext(
+	t *testing.T,
+) (
+	*ojoapp.App,
+	sdk.Context,
+	[]TestValidatorKey,
+) {
+	config := sdk.GetConfig()
+	config.SetBech32PrefixForAccount(appparams.AccountAddressPrefix, appparams.AccountPubKeyPrefix)
+	config.SetBech32PrefixForValidator(appparams.ValidatorAddressPrefix, appparams.ValidatorPubKeyPrefix)
+	config.SetBech32PrefixForConsensusNode(appparams.ConsNodeAddressPrefix, appparams.ConsNodePubKeyPrefix)
+
+	app := ojoapp.Setup(t)
+	ctx := app.BaseApp.NewContext(isCheckTx, tmproto.Header{
+		ChainID: fmt.Sprintf("test-chain-%s", tmrand.Str(4)),
+		Height:  9,
+	})
+
+	queryHelper := baseapp.NewQueryServerTestHelper(ctx, app.InterfaceRegistry())
+	oracletypes.RegisterQueryServer(queryHelper, oraclekeeper.NewQuerier(app.OracleKeeper))
+	airdroptypes.RegisterQueryServer(queryHelper, airdropkeeper.NewQuerier(app.AirdropKeeper))
+
+	sh := stakingtestutil.NewHelper(t, ctx, app.StakingKeeper)
+	sh.Denom = bondDenom
+
+	// amt := sdk.TokensFromConsensusPower(100, sdk.DefaultPowerReduction)
```

Comment:
Do we still need this commented code? And where its used and also commented in line [97](https://github.com/ojo-network/ojo/blob/0551806e2ae1866cb786ff277a078e6322cc335a/tests/integration/test_helpers.go#L97)?

Reply:
I'll just leave this for when I cleanup the test setup in the oracle module as well. There were just a few extra issues to iron out with voting power, etc.